tinycrypt/ecc.c

```diff
@@ -289,17 +289,29 @@ uECC_word_t uECC_vli_equal(const uECC_word_t *left, const uECC_word_t *right)
 	uECC_word_t tmp1, tmp2;
 	volatile int i;
 
-	for (i = NUM_ECC_WORDS - 1; i >= 0; --i) {
+	/* Start from a random location and check the correct number of iterations */
+	int flow_monitor = 0;
+	int start_offset = 1 + mbedtls_platform_random_in_range(NUM_ECC_WORDS - 1);
```

Comment:
Why not just `mbedtls_platform_random_in_range(NUM_ECC_WORDS)`?

Reply:
NUM_ECC_WORDS is equal to 8. The probability of falling 0, i.e. performing operations without starting from a random location is quite high. In this way the random value will be always in the range [1, 7], i.e. starting from a random location will be guaranteed.

Reply:
0 is also a random location :)

Reply:
If you say so, I changed it